CARET_trace/test/test_hashable_keys.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Research Institute of Systems Planning, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "caret_trace/keys_set.hpp"
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <string>
+#include <utility>
+
+TEST(HashableKeys, IntCase)
+{
+  HashableKeys<int> keys(1);
+  HashableKeys<int> keys_(1);
+  HashableKeys<int> keys__(2);
+
+  EXPECT_TRUE(keys.equal_to(keys_));
```

Comment:
It is needed to test `hashableKeys(T1 key1, T2 key2)` and `HashableKeys(T1 key1, T2 key2, T3 key3)`, isn't it?

Did you omit them because they are similar? Then it's OK.

Reply:
Thank you for pointing out.
I added: [bf33cca](https://github.com/tier4/CARET_trace/pull/46/commits/bf33ccae681c19cec15d5159ff2ed02da3fb8260)